src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -83,6 +116,33 @@ public void setNumberOfChannels(String mediaInfoValue) {
 	}
 
 	/**
+	 * Get bits per sample for this audio track.
+	 * @return bits per sample (default is 16)
+	 */
+	public int getBitsperSample() {
+		return bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track.
+	 * @param bitsperSample bits per sample to set
+	 */
+	public void setBitsperSample(int bitsperSample) {
+		if (bitsperSample < 1) {
+			throw new IllegalArgumentException("Bits per sample can't be less than 1.");
+		}
```

Comment:
I don't share you point of view ;) as usual.
I did it because i wanted avoid 0 value being set in the DIDL string, though i could also do that directly in DLNAResource, the file that i love modify :wink: 
The default values are indicated by the message in the log.

I doesn't mean that it the best way, but it was the first that come to my mind.
And also i saw the WMP use default values for bitrate and other, sometimes specific to some formats as ADTS or AAC(M4A)